bench/CMakeLists.txt

```diff
@@ -3,17 +3,32 @@ set(BENCHMARK_ENABLE_TESTING OFF CACHE BOOL "Suppressing benchmark's tests" FORC
 add_subdirectory(benchmark)
 
 # remember to add src's here
-set(benchSrcs
-    operation_tree_bench.cpp
-    main_bench.cpp
+set(BenchSources
```

Comment:
You could use a glob here to get all benchs and don't forget to add them.
Here is an example of getting all test files in other project as reference: https://github.com/SimulationEverywhere/cadmium/blob/master/CMakeLists.txt#L42